appdev/guides/pushnotifications.rst

```diff
@@ -0,0 +1,235 @@
+Push Notifications
+==================
```

Comment:
This document has all of the content I need to start using push notifications in my app. However, the layout of the content makes it feel a bit hard to follow. It is trying to be two documents, I feel:

1. How to set an app up for receiving push notifications
1. What to do with a notification when you get it

My gut feeling is that these two topics are logically separated and each requires its own section, if not its own document. What do you think?

Reply:
To add to that, the push helper can grab notifications independently of the push service, right? That makes all the more reason to have two separate documents... maybe even a chapter of documentation dedicated to notifications.

Reply:
What do you think about the latest commit with "Restructuring"? It is no divided into two topics:
"Make your app ready for Push Notifications"
"Using the push notifications API"

Reply:
Much better!